docs/user_manual/source/xsecure.rst

```diff
@@ -259,48 +267,51 @@ The OBI ([OPENHW-OBI]_) bus interfaces have associated parity and checksum signa
+  |                              |                                   |                           |                            | response phase            |
+  +------------------------------+-----------------------------------+---------------------------+----------------------------+---------------------------+
+  | ``data_gntpar_i``            | As defined in [OPENHW-OBI]_       | Always                    | When not in reset          | During data access        |
+  |                              |                                   |                           |                            | address phase             |
+  +------------------------------+-----------------------------------+---------------------------+----------------------------+---------------------------+
+  | ``data_rvalidpar_i``         | As defined in [OPENHW-OBI]_       | Always                    | When not in reset          | During data access        |
+  |                              |                                   |                           |                            | response phase            |
+  +------------------------------+-----------------------------------+---------------------------+----------------------------+---------------------------+
+  | ``instr_rchk_i``             | As defined in                     | ``cpuctrl.integrity`` = 1 | During instruction access  | During instruction access |
+  |                              | :numref:`Response phase checksum` | and PMA attributes access | response phase             | response phase            |
+  |                              |                                   | with ``integrity`` = 1    |                            |                           |
+  +------------------------------+-----------------------------------+---------------------------+----------------------------+---------------------------+
+  | ``data_rchk_i``              | As defined in                     | ``cpuctrl.integrity`` = 1 | During data access         | During data access        |
+  |                              | :numref:`Response phase checksum` | and PMA attributes access | response phase             | response phase            |
+  |                              |                                   | with ``integrity`` = 1    |                            |                           |
+  +------------------------------+-----------------------------------+---------------------------+----------------------------+---------------------------+
+
+Interface checking is performed both associated and non-associated to specific instruction execution.
+
+Non-associated interface checks are performed by only taking into account the bus interfaces themselves and use an observation interval that is as wide as
+possible (e.g. a data interface related parity or checksum error can be detected even if no load or store instruction is actually being executed).
```

Comment:
When checking rchk only during the response phase, we will not be able to detect rchk releated erros when no load or store instruction is being executed?

Reply:
You are right. Checksum checks are only enabled based on the PMA (and cpuctrl) and therefore the rchk checks are not simply based on the bus interface signals in a combinatorial manner, but also on some state indicating whether a bus response is expected to have checksums enabled or not.